docs/src/references/mappers/c8y-mapper.md

```diff
@@ -209,6 +209,44 @@ and hence there is no mapping done for main device registration messages.
 Inventory data updates for the main device are handled differently.
 :::
 
+## Auto Registration of Entities
+
+To send the telemetry/health status message to an entity, the entity has to be registered first. The entities can be registered either explicitly,
+or implicitly.
+
+To register an entity explicitly, an entity registration message has to be sent.
+
+To auto-register an entity, an explicit registration message is not required,
+but a first telemetry/health-status message that is sent on the default topic scheme (te/device/device-id/service/service-id),
+will be used to register the entity by the `c8y-mapper` and then forward the telemetry/health-status message.
+
+The auto-registration can be enabled/disabled using the `tedge` tool.
+
+The auto-registration can be disabled as below
+
+```sh
+sudo tedge config set c8y.entity_store.auto_register false
+```
+:::note
+When the auto registration is disabled, and if the device is not already registered.
+Then the c8y-mapper will not forward the incoming telemetry/health-status messages to the c8y cloud.
+But, logs/publishes an error message on the `te/errors` topic telling the entity is not registered.
+:::
+
+The auto-registration can be enabled as below
+
+```sh
+sudo tedge config set c8y.entity_store.auto_register true
+```
+Or
+
+```sh
+sudo tedge config unset c8y.entity_store.auto_register
+```
```

Comment:
```suggestion
```
I'd just drop that to reduce the verbosity, as the previous example is clear enough. Especially since it is just a boolean value with only two possible values.

Reply:
Okay, the default is `true`.